content/post/review-programs-with-graphs.md

```diff
@@ -0,0 +1,322 @@
+---
```

Comment:
You need more structure in this blog post.
It's very interesting but as I read it I have no idea where it's going.

Maybe add a quick intro telling me what you're gonna explain before jumping into the details.